src/main/java/racingcar/domain/dto/CarDTO.java

```diff
@@ -0,0 +1,18 @@
+package racingcar.domain.dto;
+
+public class CarDTO {
+  private String car;
```

Comment:
```suggestion
  private String name;
```
추천합니다. 코드 전반에 name을 쓸 곳에 car이라 되어 있는 것 같아요.